projects/vdk-plugins/vdk-meta-jobs/src/vdk/plugin/meta_jobs/remote_data_job_executor.py

```diff
@@ -0,0 +1,28 @@
+# Copyright 2021 VMware, Inc.
+# SPDX-License-Identifier: Apache-2.0
+from abc import ABC
+from typing import Dict
+
+from vdk.internal.control.configuration.vdk_config import VDKConfig
+from vdk.plugin.meta_jobs.meta import IDataJobExecutor
+from vdk.plugin.meta_jobs.remote_data_job import RemoteDataJob
+
+
+class RemoteDataJobExecutor(IDataJobExecutor):
```

Comment:
The way that you create a new job each time seems a bit odd to me. 

I would expect that you would call all methods on the same instance.
Maybe they should just be static methods if you aren't keeping the object around. 

```
job = RemoteDataJob(job_name, team_name, vdk_cfg.control_service_rest_api_url)
job.start_job_execution()
job.get_job_execution_status(execution_id)
```

Reply:
Don't focus on the implementation but on the interface

I plan (in the far future!) to refactor the way it's passed configuration. 

VDKConfig is the way vdk-control-cli keeps and passes configuration while vdk-core uses ConfigurationBuilder and vdk_configure plugins. 

I want to streamline it . vdk-control-cli was created before vdk-core and plugins come into place that's why it's different. Iti won't be soon and not in this change.

Reply:
Does that make sense?